src/nvim/highlight_defs.h

```diff
@@ -0,0 +1,126 @@
+#ifndef NVIM_HIGHLIGHT_DEFS_H
+#define NVIM_HIGHLIGHT_DEFS_H
```

Comment:
should we eventually move highlight logic to highlight.c? (I haven't looked to see if this makes sense, just wondering if you already thought about it)